crates/hc_bundle/src/error.rs

```diff
@@ -42,6 +42,9 @@ pub enum HcBundleError {
 
     #[error(transparent)]
     ModuleCompileError(#[from] CompileError),
+
+    #[error("Requires hc_bundle to be built with the feature flag: {0}")]
+    UnsupportedFeatureError(String)
```

Comment:
Not sure if this is idiomatic or I should instead panic with the `unimplemented` macro.